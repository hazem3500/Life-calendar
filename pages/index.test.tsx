import renderer from 'react-test-renderer'
import IndexPage from '.'

describe('Index page', () => {
    it('should match the snapshot', () => {
    const launch = {
      timestamp: 1605451340000,
      mission: 'Mission Name',
      site: 'Kennedy Space Center',
      rocket: 'Falcon 9',
    }
    const tree = renderer.create(<IndexPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
