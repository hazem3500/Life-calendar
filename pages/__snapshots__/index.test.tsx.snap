// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<main>
  <h1>
    Next SpaceX Launch: 
    Mission Name
  </h1>
  <p>
    Falcon 9
     will take off from 
    Kennedy Space Center
     on 
    Sun Nov 15 2020
  </p>
</main>
`;
